### ISSUE TITLE
您好 想请问下 bert_graph.py 下面第50行

### ISSUE THREAD

**Issue by jackychancjcjcj** [wodejiafeiyu/ccks2021-track3-top1#2]
outputs = (logits,) + outputs[2:] 这个outputs[2:]的含义是啥呀，是hidden_states和attention嘛？和logits的维度对不上呀，希望能解答下！

**Comment by wodejiafeiyu** [comment]
你好，outputs[2:]代表的是 embeding的输出加上后面12层的输出，总共13层，维度都是 batchsize，seq_len ,hidden_dim
